packages/kit/src/core/adapt/builder.js

```diff
@@ -46,7 +46,7 @@ export function create_builder({ config, build_data, routes, prerendered, log })
 
 				return {
 					id: route.id,
-					type: route.page ? 'page' : 'endpoint', // TODO change this if support pages+endpoints
+					type: route.page && route.endpoint ? 'both' : route.page ? 'page' : 'endpoint',
```

Comment:
It looks like this was never used by one of our adapters. Adding a third option is non-breaking, so any community adapters that _might_ use this can adapt

Reply:
I checked all the community adapters, not one of them is using this. I think we can safely remove it altogether

```suggestion
```